ui/decredmaterial/linearlayout.go

```diff
@@ -25,6 +25,10 @@ type LinearLayout struct {
 	Direction   layout.Direction
 }
 
+func (ll LinearLayout) Layout2(gtx C, wdg layout.Widget) D {
```

Comment:
Can we find a better name for this? If we can't how's it different from the Layout method, can we add a comment to it?